templates/zammad/1/docker-compose.yml

```diff
@@ -0,0 +1,81 @@
+version: '2'
+
+services:
+
+  zammad:
+    depends_on:
+      - zammad-elasticsearch
+      - zammad-postgresql
+    image: zammad/zammad-docker-compose:zammad
+    labels:
+      io.rancher.container.pull_image: always
+      io.rancher.container.start_once: 'true'
+    restart: on-failure
+    volumes:
+      - data-zammad:/home/zammad
+
+  zammad-elasticsearch:
+    image: zammad/zammad-docker-compose:zammad-elasticsearch
+    labels:
+      io.rancher.container.pull_image: always
+    restart: always
+
+  zammad-nginx:
+    depends_on:
+      - zammad
+    image: zammad/zammad-docker-compose:zammad-nginx
+    labels:
+      io.rancher.container.pull_image: always
+    links:
+      - zammad-railsserver
+      - zammad-websocket
+    ports:
+      - "80:80"
```

Comment:
Not necessary, it's load balancer responsability ;)

Reply:
its removed now.

Reply:
Just removed at version 1.5, `templates/zammad/3/docker-compose.yml`, but not added load balancer. No service published.